packages/gestalt-codemods/next/__testfixtures__/ternary-color.input.js

```diff
@@ -0,0 +1,8 @@
+// @flow strict
+import { Toast } from 'gestalt';
+
+export default function TestComponent() {
+  return (
+    <Toast color={true ? 'red' : 'white'} text="Toasty McToasterFace" />
```

Comment:
you might need to set true inside a const

Reply:
I don't think that should matter for this codemod, since I prompt manual conversion for any ternary regardless of how the condition is presented